.github/workflows/frontend_lint.yaml

```diff
@@ -0,0 +1,22 @@
+name: Lint Frontend Code
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
```

Comment:
nit: should this have a '-' in front of it? I'm not sure of yaml styling but the 'uses: ' line on 14 has a dash in front of it

Reply:
Yeah YAML is really odd. The `-` actually means "start a new item in this list", so the first step has only a `uses` field on it. The second step has `name` and `uses`.